0006/README.md

```diff
@@ -206,6 +206,31 @@ Git branch is the way to manage versioned specifications.
     `/docs/github-actions-workflow/<version>.md`. All specification files in the
     same branch share the same version.
 
+#### Code Contribution
+
+Since the repo uses the Git submodule in a non-standard way and for easy
+building website purposes, people SHOULD NOT use submodules to commit any
+change.
+
+When a community member needs to make a contribution, they fork the repo (and
+clone it locally if they are familiar with git). If the change is for a
+versioned file (a.k.a. spec file), they should make the change in the version
+branch, and send a PR to the version branch. Meanwhile, if the change is for an
+unversioned file or the website itself, they should make the change in `main`,
+and send a PR to `main`.
+
+However, if the change comes from one of the repo owners, and it has to modify
+files in both `main` and version branches in one PR, we assume the owner
+understands the risk and knows how to send a proper code update with Submodule
+changes.
+
+#### Submodule Pointer
+
+A new change in a version branch makes the Submodule pointer in the `main`
+branch stale. A Github Workflow script may be created to automatically update
+the Submodule pointing to the latest commit, when the PR is merged. If desired,
```

Comment:
Can we make this part of the plan, to have a workflow that automatically submits a change to update the submodule? I don't think anyone will have the rigor to do it manually.

Reply:
I will wait and cleanup this section after our discussion :)

Reply:
I added a bulletin to talk about sync a git subtree change to its branch.